core/src/main/java/com/sequenceiq/cloudbreak/controller/ManagementPackController.java

```diff
@@ -114,15 +129,19 @@ private ManagementPackResponse createMpack(IdentityUser user, ManagementPackRequ
     }
 
     private ManagementPackResponse post(ManagementPackRequest mpackRequest) {
-        IdentityUser user = authenticatedUserService.getCbUser();
+        User user = userService.getOrCreate(restRequestThreadLocalService.getIdentityUser());
         return createMpack(user, mpackRequest);
     }
 
     private Set<ManagementPackResponse> getAll() {
-        return toJsonList(mpackService.findAllForUsersDefaultOrganization());
+        User user = userService.getOrCreate(restRequestThreadLocalService.getIdentityUser());
```

Comment:
Now this 2 lines of code is present everywhere. 
```
User user = userService.getOrCreate(restRequestThreadLocalService.getIdentityUser());
Organization organization = organizationService.get(restRequestThreadLocalService.getRequestedOrgId(), user);
```

I think methods with default organization should be re-introduced because this is a _big_ cohesion problem in the code. These methods could go to a separate interface and used only by V1 controllers, thus we avoid misusage.